mirgecom/simutil.py

```diff
@@ -66,12 +66,16 @@ def inviscid_sim_timestep(discr, state, t, dt, cfl, eos,
                           t_final, constant_cfl=False):
     """Return the maximum stable dt."""
     mydt = dt
+    dt_left = t_final - t
```

Comment:
```suggestion
    t_remaining = t_final - t
```
? I was briefly confused what this had to do with left/right.